package.json

```diff
@@ -17,17 +20,23 @@
     "react-dom": "17.0.2"
   },
   "devDependencies": {
+    "@testing-library/react": "^12.1.4",
     "@types/node": "17.0.21",
     "@types/react": "17.0.39",
     "@typescript-eslint/eslint-plugin": "^5.14.0",
     "@typescript-eslint/parser": "^5.14.0",
+    "@vitejs/plugin-react": "^1.3.2",
+    "@vitest/ui": "^0.12.4",
```

Comment:
Went with VI to try out a new testing framework which has support for testing syntax I'm used to while getting a better dev experience. 

I especially like the testing watcher, which has a console and browser view.

![image](https://user-images.githubusercontent.com/667983/187048358-6b3c21bc-459d-49b5-88f7-36557f18ab64.png)